bin/run_robot_cron

```diff
@@ -17,7 +17,7 @@ timestamp = DateTime.now.strftime('%Y%m%d-%H%M%S')
 
 # See if the robot is already running.
 # This list is going to detect this process.
-procs_running = `ps -ef | grep #{robot_name} | grep -v 'grep' | wc -l`.strip.to_i
+procs_running = `ps -ef | grep #{robot_name} | grep -v 'grep' | grep -v 'run_robot_cron' | wc -l`.strip.to_i
```

Comment:
I think this completely defeats the purpose of how this looks for duplicates, right?

Reply:
Perhaps, but this seems to blocking on the command that is being run, not the actual process....

Reply:
I'm also happy to have this held for the day while I work on locking, but would like the option of getting _something_ in place this evening otherwise.

Reply:
Just comment out everything here? Then it's clear that this is a nop.